api/client.go

```diff
@@ -33,14 +40,34 @@ func checkError(resp *http.Response, body []byte) error {
 	return apiError
 }
 
+// Host returns the default host to use for the client. It is determined in the following order:
+// 1. The OLLAMA_CLIENT_HOST environment variable
+// 2. The default host (localhost:11434)
+func Host() string {
```

Comment:
I think `Host()` is a bit too generic. What if other host-related functions are added later? I think `GetHost()` would be more specific and descriptive.